<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>1251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image DSP</string>
  </property>
  <widget class="QDockWidget" name="inputImgViewDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Input Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="inputImgViewDockCentralWidget">
    <layout class="QVBoxLayout" name="inputImgViewDockLayout">
     <item>
      <layout class="QVBoxLayout" name="inputImgViewLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputImgViewDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Output Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="outputImgViewDockCentralWidget">
    <layout class="QVBoxLayout" name="imgViewDockLayout">
     <item>
      <layout class="QVBoxLayout" name="outputImgViewLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="progListDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Programs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="progListDockCentralWidget">
    <layout class="QVBoxLayout" name="progListDockLayout">
     <item>
      <widget class="QListView" name="progList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="paramsListDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="paramsListDockCentralWidget">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="textureListLayout">
     <item>
      <layout class="QVBoxLayout" name="paramsListLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="inImgsListDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Input Images</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="inImgsListDockCentralWidget">
    <layout class="QVBoxLayout" name="inImgsListDockLayout">
     <item>
      <widget class="QListView" name="inImgsList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outImgsListDock">
   <property name="windowTitle">
    <string>Output Images</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="outImgsListDockCentralWidget">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QPushButton" name="saveOutImg">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="outImageText"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="browseOutImg">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImportGeometry">
   <property name="text">
    <string>&amp;Geometry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionImportTexture">
   <property name="text">
    <string>&amp;Texture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExportGeometry">
   <property name="text">
    <string>&amp;Geometry</string>
   </property>
  </action>
  <action name="actionExportTexture">
   <property name="text">
    <string>&amp;Texture</string>
   </property>
  </action>
  <action name="actionTextureName">
   <property name="text">
    <string>Texture Name</string>
   </property>
  </action>
  <action name="actionTextureDescription">
   <property name="text">
    <string>Texture Description</string>
   </property>
  </action>
  <action name="actionDeleteTexture">
   <property name="text">
    <string>D&amp;elete Texture</string>
   </property>
  </action>
  <action name="actionResetModelCamera">
   <property name="text">
    <string>Reset &amp;Model Camera</string>
   </property>
  </action>
  <action name="actionResetUVCamera">
   <property name="text">
    <string>Reset &amp;UV Camera</string>
   </property>
  </action>
  <action name="actionResetAllCameras">
   <property name="text">
    <string>Reset &amp;All Cameras</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>&amp;Language</string>
   </property>
  </action>
  <action name="actionTextureItemAlwaysEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Texture Item Always Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
